### ISSUE TITLE
ACME integration

### ISSUE THREAD

**Issue by Geal** [sozu-proxy/sozu#686]
there's an external tool called [sozu-acme](https://github.com/sozu-proxy/sozu-acme) that can set up certificates in sozu. It works well but must be run manually.

Now that sozu has a better futures executor in the main process, I'd like to integrate more tasks like this.
For large deployments like we do at Clever Cloud, this would be deactivated (because certificates would better be managed centrally instead of in every load balancer), but for smaller architectures, I'd like to have sozu automatically set up certificates and HTTP->HTTPS redirections, to make the easy path safer

**Comment by stappersg** [comment]
This issue should be closed:

* There is #926 as "successor"
* https://github.com/sozu-proxy/sozu-acme/blob/master/README.md has _"This repository has been integrated into the main Sōzu repository. It is available directly from the command line of Sōzu."_
* https://github.com/sozu-proxy/sozu-acme is marked _"This repository has been archived by the owner on Dec 8, 2022. It is now read-only."_
* Dangling issues drain human energy.

**Comment by Geal** [comment]
I've left the company and stopped contributing regularly to this project 2 years ago, so sorry if my issue triaging has not been great lately.
Why do you feel it is draining here? Are you looking to contribute to this project? If yes, it might be best to start by the docs and try to run it, and contact the developers directly, because the issues are heavily influenced by Clever Cloud's internal roadmap

**Comment by stappersg** [comment]
On Sat, Aug 05, 2023 at 10:24:39AM -0700, Geoffroy Couprie wrote:
> Why do you feel it is draining here?

It emmits "zero fuck given" or in less harsh words
"we don't care about it, neither should you do"


> Are you looking to contribute to this project? If yes, it might be
> best to start by the docs and try to run it,

Work in progress, part of it is checking open issues  ;-)


> and contact the developers directly,
> because the issues are heavily influenced by Clever Cloud's internal roadmap

Acknowledge.  FWIW  I tent to aim for win/win.


> ... stopped contributing regularly to this project ...

Another reason for closing this issue.


Groeten
Geert Stappers
-- 
Silence is hard to parse

**Comment by FlorentinDUBOIS** [comment]
Hello @stappersg, 
Could you give me more details about what you find confusing?
What are you trying to acheive?

**Comment by stappersg** [comment]
> What are you trying to achieve?

Less open issues


> more details about what you find confusing

#926 superseeds #686

**Comment by Keksoj** [comment]
This issue is indeed superseded by #926
